group_vars/datagrid.yml

```diff
@@ -17,7 +17,7 @@ jdg_db_pass: keycloak-pass
 
 jdg_default_realm_tls: True
 jdg_keystore_path: /etc/pki/java/keystore.jks
-jdg_keystore_alias: "*.rhssocrossdc.com"
+jdg_keystore_alias: "*.internal.cloudapp.net"
```

Comment:
pls reuse the domain_name variable defined in group_vars/all.yml here

Reply:
Changes added, thank you for pointing out.